### ISSUE TITLE
Cannot handle grpc gateway options

### ISSUE THREAD

**Issue by nonpcnpc** [tallstoat/pbparser#6]
It looks like the library cannot handle rpc options - specifically I am using the grpc gateway annotations:
```
rpc CreateWorkspace(CreateWorkspaceRequest) returns (CreateWorkspaceResponse) {
        option (google.api.http) = { post: "/v1/workspaces" body: "*" };
}
```

also it will complain about external dependencies(imports) which I'm not interested in.

**Comment by tallstoat** [comment]
@nonpcnpc Sorry about not getting to this earlier. 

Will take care of the grpc annotations parsing so the parser won't break. The parser will however not try to parse the option value but just return the content between the curly braces. This is to avoid getting into grpc specific parsing. It can however be done outside of this parse with the option value which is returned by the invoking code.  

About the external dependencies (imports), I'll look into providing a flag using which invokers can choose to instruct the parser not to complain about such imports.

**Comment by tallstoat** [comment]
https://github.com/tallstoat/pbparser/issues/7 is opened for the enhancement part of this ticket.

**Comment by tallstoat** [comment]
This is now fixed in 0.3.0 branch. Closing.
